server/base/src/main/java/org/apache/accumulo/server/util/FileSystemMonitor.java

```diff
@@ -121,8 +121,8 @@ public FileSystemMonitor(final String procFile, long period, AccumuloConfigurati
             try {
               checkMount(mount);
             } catch (final Exception e) {
-              Halt.halt(-42,
-                  () -> log.error("Exception while checking mount points, halting process", e));
+              Halt.halt(-42, "Exception while checking mount points, halting process",
```

Comment:
Negative 42 is a little crazy. I'm not sure where these numbers come from. But, we really shouldn't be using negative values for exit codes. Some environments don't support that. Positive 42 is less crazy (but still strange and arbitrary).